exercises/space-age/.meta/gen.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"log"
+	"text/template"
+
+	"../../../gen"
+	space "github.com/exercism/go/exercises/space-age"
+)
```

Comment:
Recommend this change:
```
-       space "github.com/exercism/go/exercises/space-age"
+       space "../../space-age"
```
I think it best to keep the path to the dependent package relative, just as the path to the 'gen' package is relative.

Reply:
Done.